src/planner/CompositePlanner.cpp

```diff
@@ -0,0 +1,43 @@
+#include "aikido/planner/CompositePlanner.hpp"
+
+#include <algorithm>
+#include <utility>
+
+namespace aikido {
+namespace planner {
+
+//==============================================================================
+CompositePlanner::CompositePlanner(
+    statespace::ConstStateSpacePtr stateSpace,
+    const std::vector<PlannerPtr>& planners)
+  : Planner(std::move(stateSpace)), mPlanners(planners)
+{
+  // Do nothing
+}
+
+//==============================================================================
+bool CompositePlanner::hasPlanner(const Planner* planner) const
```

Comment:
Could it be more useful to do a type check instead of reference check?

Reply:
This class is general to handle the case of having different planner instances of the same type. In that case, type checking will return the first planner instance of the type. Or we could have another function that returns set of planners of the type.

As a side note, it'd be better to take type for type checking.